apps/web/components/layout/footer/footer.tsx

```diff
@@ -14,7 +14,7 @@ const Footer = () => {
 						className="text-primary dark:text-gray-300"
 						rel="noreferrer"
 					>
-						{t('layout.footer.COPY_RIGHT4')}
+						{process.env.NEXT_PUBLIC_APP_NAME}
```

Comment:
@samuelmbabhazi we should be using something from NextJs config, not use process.env in components directly!?